src/test/java/uk/gov/pay/directdebit/webhook/gocardless/services/WebhookGoCardlessServiceTest.java

```diff
@@ -66,63 +60,24 @@
 
     @Test
     public void shouldInsertEvents() {
-        GoCardlessEvent firstGoCardlessEvent = aGoCardlessEventFixture().toEntity();
-        GoCardlessEvent secondGoCardlessEvent = aGoCardlessEventFixture().toEntity();
-        List<GoCardlessEvent> events = Arrays.asList(firstGoCardlessEvent, secondGoCardlessEvent);
-        webhookGoCardlessService.handleEvents(events);
-        verify(mockedGoCardlessEventService).storeEvent(firstGoCardlessEvent);
-        verify(mockedGoCardlessEventService).storeEvent(secondGoCardlessEvent);
-    }
-
-    @Test
-    public void shouldStoreButNotHandleEventsWithRefundsResource() {
-        GoCardlessEvent goCardlessEvent = aGoCardlessEventFixture().withResourceType(REFUNDS).withAction("created").toEntity();
-
-        List<GoCardlessEvent> events = Collections.singletonList(goCardlessEvent);
-        webhookGoCardlessService.handleEvents(events);
-        verify(mockedGoCardlessEventService).storeEvent(goCardlessEvent);
-    }
-
-    @Test
-    public void shouldStoreButNotHandleEventsWithSubscriptionsResource() {
-        GoCardlessEvent goCardlessEvent = aGoCardlessEventFixture().withResourceType(SUBSCRIPTIONS).withAction("created").toEntity();
-
-        List<GoCardlessEvent> events = Collections.singletonList(goCardlessEvent);
-        webhookGoCardlessService.handleEvents(events);
-        verify(mockedGoCardlessEventService).storeEvent(goCardlessEvent);
+        GoCardlessEvent first = aGoCardlessEventFixture().withResourceType(GoCardlessResourceType.REFUNDS).toEntity();
```

Comment:
I suppose this test could check events for each of the five resources are stored correctly (maybe using a parameterised test?).

Reply:
fixed